migrations/postgres/20231223165738_user_profiles.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE entity ADD display_name VARCHAR(35);
```

Comment:
Whereas SQLite doesn't support adding multiple columns in a single `ALTER`, I believe Postgres does.